docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>have you tried plotting the uncertainty sausages in the same plot as the measurents? Would be good to see if their width is appropriate or not</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>They _are_plotted in every plot. For the components that are observed, the uncertainty is just lower (above row of plots)</p><p> </p>

Reply:
<p>ah okay my bad, I did not see that. </p>